content/applications/services/helpdesk/overview/sla.rst

```diff
@@ -2,183 +2,209 @@
-   --> Employees --> Work Organization`.
+Create a new SLA policy
+=======================
 
-To create a new policy, click the smart button on the team`s settings page or go to
-:menuselection:`Helpdesk -->  Configuration --> SLA Policies`, and click :guilabel:`New`. Start by
-entering a :guilabel:`Title` and a :guilabel:`Description` for the new policy, and proceed to fill
-out the form using the steps below.
+To create a new policy, go to :menuselection:`Helpdesk app --> Configuration --> SLA Policies`, and
+click :guilabel:`New`. Alternatively, go to :menuselection:`Helpdesk app --> Configuration -->
+Helpdesk Teams` and click on a team. Then, click the :guilabel:`SLA Policies` smart button at the
+top of the team's settings page, then click :guilabel:`New`.
+
+On the blank :abbr:`SLA (Service Level Agreement)` policy record, enter a :guilabel:`Title` and a
+:guilabel:`Description` for the new policy, and proceed to fill out the form using the steps below.
 
 Define the criteria for an SLA policy
 -------------------------------------
 
-The :guilabel:`Criteria` section is used to identify what tickets this policy will be applied to.
+The :guilabel:`Criteria` section is used to identify which tickets this policy is applied to.
+
 Fill out the following fields to adjust the selection criteria:
 
+.. note::
+   Unless otherwise indicated, multiple selections can be made for each field.
+
 - :guilabel:`Team`: a policy can only be applied to one team. *This field is required.*
-- :guilabel:`Priority`: the priority level for a ticket is identified by selecting the number of
-  stars representing the priority level on the kanban card or the ticket itself. The :abbr:`SLA
-  (Service Level Agreement)` will only be applied once the priority level has been updated on the
-  ticket to match the :abbr:`SLA (Service Level Agreement)` criteria. If no selection is made in
-  this field, this policy will only apply to tickets marked as `Low Priority` (zero stars).
-- :guilabel:`Types`: ticket types can be helpful when indicating when a ticket is a
-  customer question, that can be solved with a quick response, or an issue, that may require
-  additional investigation. Multiple ticket types can be selected for this field. If no
-  selection is made, this policy will apply to all ticket types.
-- :guilabel:`Tags`: tags are applied to briefly indicate what the ticket is about. Multiple tags
-  can be applied to a single ticket.
+- :guilabel:`Priority`: the priority level for a ticket is identified by selecting one or more of
+  the :guilabel:`⭐ (star)` icons representing the priority level on the kanban card or the ticket
+  itself. The :abbr:`SLA (Service Level Agreement)` will **only** be applied after the priority
+  level has been updated on the ticket to match the :abbr:`SLA (Service Level Agreement)` criteria.
+  If no selection is made in this field, this policy will only apply to tickets marked as `Low
+  Priority`, meaning those with zero stars.
```

Comment:
`Type` was removed as a field/ticket identifier in 17.1. Essentially, adding a ticket type was another way to differentiate between tickets, but it operated almost the exact same way as `tags`, and it was deemed unnecessary. I didn't include it to keep the doc as up-to-date as possible and also because it doesn't have an impact on any functionality.